### DISCUSSION TITLE
Jump to other side of selection?

### DISCUSSION THREAD

**Discussion by bellwether-softworks** [helix-editor/helix#6090] [Q&A] [answered]
I'm looking for the equivalent of pressing `o` in vim's select mode, in which your cursor bounces to the opposite end of the selection.  Does this command, or some functional equivalent, exist in helix?

**Comment by gabydd** [comment, marked as answer]
Yeah this can be done with ``Alt-;`` which flips the anchor and the head of the selection, in helix a selection has a head and an anchor the head is basically the cursor so when the cursor moves so does the head of the selection

**Comment by n8henrie** [comment]
Possibly helpful for other searchers -- in the commands palette, this is called `flip_selections` (I don't think you can currently search for e.g. `Alt-;` or `<A-;>`).

**Reply by xpe** [reply]
After pressing <kbd>space ?</kbd>, you can search for keybindings by starting with `%b ` e.g.: `%b A-;`. 
<img width="848" height="112" alt="image" src="https://github.com/user-attachments/assets/2cad88aa-2e0e-4e7a-82b7-6d1d4de00261" />

Also, the `%b` isn't the only special search syntax, nor is it limited to the help picker, which is just one [picker](https://helix-editor.vercel.app/usage/pickers) of many. See: [Filtering Picker Results](https://docs.helix-editor.com/pickers.html#filtering-picker-results):

> If a picker shows multiple columns, you may apply the filter to a specific column by prefixing the column name with `%`. Column names can be shortened to any prefix, so `%p`, `%pa` or `%pat` all mean the same as `%path`. For example, a query of helix `%p .toml !lang` in the global search picker searches for the term "helix" within files with paths ending in ".toml" but not including "lang".
